content/source/docs/cloud/workspaces/vcs.html.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: "cloud"
+page_title: "VCS Connections - Workspaces - Terraform Cloud"
+---
+
+# Configuring Workspace VCS Connections
+
+Any Terraform Cloud workspace can be connected to a version control system (VCS) repository that contains its Terraform configuration. You can assign a connection [when creating a workspace](./creating.html), and can modify that connection in the "Version Control" page of the workspace's settings.
+
+For more information about how configuration versions and connected repositories work, see [Terraform Configurations in Terraform Cloud Workspaces](./configurations.html).
+
+-> **API:** See the [Update a Workspace endpoint](../api/workspaces.html#update-a-workspace) (`PATCH /organizations/:organization_name/workspaces/:name`).
```

Comment:
Remove or move to the bottom of this section.

Reply:
(tabling)